tools/tm-signer-harness/main.go

```diff
@@ -147,6 +150,17 @@ func extractKey(tmhome, outputPath string) {
 	logger.Info("Successfully wrote private key", "output", outputPath)
 }
 
+func setDefaultHome() {
```

Comment:
Why not just use `init()`?

Reply:
Sure. was thinking less complicating in `init()`